CMakeLists.txt

```diff
@@ -150,7 +150,10 @@ add_subdirectory(lib)
 add_subdirectory(exe)
 
 # Add test executables for unit testing
-add_test(NAME "library\\ tests" COMMAND libfacter_test)
+add_test(NAME "libfacter\\ tests" COMMAND libfacter_test)
+if (RUBY_FOUND)
+    add_test(NAME "libfacter\\ specs" COMMAND bundle exec rspec -I ${CMAKE_BINARY_DIR}/lib WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/lib")
```

Comment:
You may need to quote this. `libfacter_test` and `facter` resolve to targets, so they can identify the test path.